src/Layout/redist/redist.csproj

```diff
@@ -48,6 +48,7 @@
                       ReferenceOutputAssembly="false" />
     <ProjectReference Include="..\..\Tasks\Microsoft.NET.Build.Extensions.Tasks\Microsoft.NET.Build.Extensions.Tasks.csproj"
                       ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\..\Resolvers\Microsoft.NET.Sdk.WorkloadResolver\Microsoft.NET.Sdk.WorkloadResolver.csproj" />
```

Comment:
why this is needed? It is not part of existing msbuild resolver's dependency?

Reply:
The workload resolver is combined with the SDK resolver for the full MSBuild resolver.  However, that resolver isn't used at all in .NET Core MSBuild, as there it doesn't need to go through the global.json resolution.  So the workload resolver is still in a separate, new MSBuild resolver DLL for .NET Core.

Reply:
Could you test both on commandline and in VS? I want to make sure it can boot up giving the complexity in dlls